docs/sql-guide/functions/function-tuple.md

```diff
@@ -7,85 +7,39 @@ grand_parent: SQL guide
 
 # TUPLE() function
 
-The `TUPLE()` function takes in n arguments and returns a tuple containing an element for each argument.
+`TUPLE()` is a mathematical function that returns an ordered list of values corresponding to and derived from a supplied argument.
 
 ## Syntax
 
+```sql
+TUPLE (
+  [ordinal_position,...] |
+  [column_name,...]
+  )
 ```
+<!-- original syntax
```

Comment:
Need an opinion on this please; it's derived from the example SQL
If correct, the original syntax can be removed